helm/appstore-metadata-service/templates/ingress.yaml

```diff
@@ -0,0 +1,83 @@
+#
+# If not stated otherwise in this file or this component's LICENSE file the
+# following copyright and licenses apply:
+#
+# Copyright 2021 Liberty Global Technology Services BV
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "appstore-metadata-charts.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ include "appstore-metadata-charts.fullname" . }}
+  annotations:
+    kubernetes.io/ingress.class: payload
+spec:
+  rules:
+  - host: {{ include "appstore-metadata-charts.fullname" . }}.{{ .Release.Namespace }}.{{ .Values.ingress.domainName }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: {{ include "appstore-metadata-charts.fullname" . }}
+          servicePort: 8080
+{{- if eq .Values.addIngressesAndSecretsForSTBAndAS3 "true" }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "appstore-metadata-charts.fullname" . }}-stb
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ include "appstore-metadata-charts.fullname" . }}-stb
+  annotations:
+    kubernetes.io/ingress.class: payload
+    ingress.kubernetes.io/auth-type: basic
+    ingress.kubernetes.io/auth-secret: stb-basic-auth
+    ingress.kubernetes.io/auth-realm: "Authentication Required - ok"
+spec:
+  rules:
+  - host: {{ include "appstore-metadata-charts.fullname" . }}-stb.{{ .Release.Namespace }}.{{ .Values.ingress.domainName }}
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: {{ include "appstore-metadata-charts.fullname" . }}
+          servicePort: 8080
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "appstore-metadata-charts.fullname" . }}-as3
```

Comment:
`as3` This is a bit meaningless as references a non-existing service.
In the past this was called `maintainers` context - as it was referencing something which is much clearer,
as it says clearly who can access this part.

Reply:
after discussing with the Component Owner we decided to leave it as it is